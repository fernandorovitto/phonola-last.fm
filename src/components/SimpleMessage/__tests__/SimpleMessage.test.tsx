import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import SimpleMessage from '../SimpleMessage';

describe('Test for SimpleMessage Component', () => {
  test('Should render the component', () => {
    const tree = renderer.create(<SimpleMessage text={'TEST'} />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
