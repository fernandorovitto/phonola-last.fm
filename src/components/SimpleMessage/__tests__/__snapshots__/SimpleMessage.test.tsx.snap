// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for SimpleMessage Component Should render the component 1`] = `
.c0 {
  background-color: black;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  opacity: 0.5;
  width: 100%;
}

.c1 {
  font-size: 18px;
  margin: 20px 0;
}

<div
  className="c0"
>
  <p
    className="c1"
  >
    TEST
  </p>
</div>
`;
